notebooks/120-tf2-object-detection-to-openvino/120-tf2-object-detection-to-openvino.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<p>can we put some description about input  and outputs in notebook too (without it looks like some context for interpreting model results is missing in notebook and without following the link it is not self-sufficient</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Added description of model input and outputs